docs/rules/no-dynamic-require.md

```diff
@@ -18,7 +18,5 @@ require(name());
 ### Pass
 
 ```js
-require('../name');
```

Comment:
True, but it seems like it is duplicated a bit below.
@yooungt13 If you could, instead of removing it, replace it by a library import (like `lodash`), that might be better for the purpose of the example :)